src/W2.EntityFrameworkCore/Migrations/20241058023402_W2PermissionData.cs

```diff
@@ -0,0 +1,275 @@
+                    ('33fc5382-9f93-41d6-acc7-751e239b7e4f', 'View List Tasks', 'Tasks.View', '881263f8-95d7-4410-88e5-d3873f99b53e', NULL, '2024-11-19T07:35:52.986298Z'),
+                    ('6c131f7f-0295-417d-8472-b792ec8250ee', 'Update Task Status', 'Tasks.UpdateStatus', '881263f8-95d7-4410-88e5-d3873f99b53e', NULL, '2024-11-19T07:35:52.986557Z'),
+                    ('cdfae99c-e8f7-4a0a-9802-401cb61bc4c7', 'View All Tasks', 'Tasks.ViewAll', '881263f8-95d7-4410-88e5-d3873f99b53e', NULL, '2024-11-19T07:35:52.986430Z'),
+
+                    -- Children Permissions for Users
+                    ('d51e853a-4d25-4121-bec1-c4a235b3db3e', 'Users Management', 'Users', NULL, NULL, '2024-11-19T07:35:52.981763Z'),
+                    ('3686a277-2718-4f19-b061-b59ac015f786', 'View List Users', 'Users.View', 'd51e853a-4d25-4121-bec1-c4a235b3db3e', NULL, '2024-11-19T07:35:52.986832Z'),
+                    ('bf6a0114-08a8-49ec-99d2-b12b35ae71ae', 'Update User', 'Users.Update', 'd51e853a-4d25-4121-bec1-c4a235b3db3e', NULL, '2024-11-19T07:35:52.986971Z'),
+
+                    -- Children Permissions for Settings
+                    ('e8d8e99d-d6e4-4199-823e-67bef350aa1a', 'Settings Management', 'Settings', NULL, NULL, '2024-11-19T07:35:52.981898Z'),
+                    ('0865f67b-f075-4107-9044-a9c7ea9112be', 'View List Settings', 'Settings.View', 'e8d8e99d-d6e4-4199-823e-67bef350aa1a', NULL, '2024-11-19T07:35:52.987104Z'),
+                    ('43f96ebf-ebce-4927-894f-14db8232b7e6', 'Delete Setting', 'Settings.Delete', 'e8d8e99d-d6e4-4199-823e-67bef350aa1a', NULL, '2024-11-19T07:35:52.987565Z'),
+                    ('46c5ce9f-3c2e-4462-aeea-fed83c9fd1cc', 'Create Setting', 'Settings.Create', 'e8d8e99d-d6e4-4199-823e-67bef350aa1a', NULL, '2024-11-19T07:35:52.987235Z'),
+                    ('79931735-d83f-4799-b1df-36932923ba33', 'Update Setting', 'Settings.Update', 'e8d8e99d-d6e4-4199-823e-67bef350aa1a', NULL, '2024-11-19T07:35:52.987426Z'),
+
+                    -- Children Permissions for WorkflowDefinitions
+                    ('6734d713-cd5c-4c9d-880f-dc0e3955d8b1', 'Request Templates', 'WorkflowDefinitions', NULL, NULL, '2024-11-19T07:35:52.973059Z'),
+                    ('643d855c-8846-4586-aeb4-3299a72e2813', 'Import Request Templates', 'WorkflowDefinitions.Import', '6734d713-cd5c-4c9d-880f-dc0e3955d8b1', NULL, '2024-11-19T07:35:52.985121Z'),
+                    ('6dc377be-198f-4a81-869d-b1c8666d2706', 'Edit Request Templates', 'WorkflowDefinitions.Edit', '6734d713-cd5c-4c9d-880f-dc0e3955d8b1', NULL, '2024-11-19T07:35:52.985892Z'),
+                    ('9cefeb05-62cf-4c58-8157-5f6c82fbe9e1', 'Create Request Templates', 'WorkflowDefinitions.Create', '6734d713-cd5c-4c9d-880f-dc0e3955d8b1', NULL, '2024-11-19T07:35:52.985389Z');
+            ");
+            migrationBuilder.Sql(@"
```

Comment:
1. AbpRoles trường Permissions chỉ lưu permissionIDs (array permission id) thôi không lưu cái cục object to đùng kia
2. query lại bảng W2Permissions, roi lay all id add vào.